packages/components/src/mobile/inserter-button/index.native.js

```diff
@@ -6,115 +6,107 @@ import { View, TouchableHighlight, Text } from 'react-native';
 /**
  * WordPress dependencies
  */
-import { Component } from '@wordpress/element';
+import { useCallback } from '@wordpress/element';
 import { Icon } from '@wordpress/components';
 import { withPreferredColorScheme } from '@wordpress/compose';
 import { __, sprintf } from '@wordpress/i18n';
 import { sparkles } from '@wordpress/icons';
+import { useDispatch } from '@wordpress/data';
+import { store as noticesStore } from '@wordpress/notices';
 
 /**
  * Internal dependencies
  */
 import styles from './style.scss';
 
-class MenuItem extends Component {
-	constructor() {
-		super( ...arguments );
+function MenuItemFunction( {
```

Comment:
What is your rationale for adding `Function` to the name here? Is there a spec or similar style guideline being used elsewhere that inspired this decision?

Reply:
Oh no, that's my bad. I named this function while refactoring from the class Component to make sure I didn't miss anything and could test both at the same time. And I forgot to rename it back.